account/user.go

```diff
@@ -115,6 +118,18 @@ func (m *GormUserRepository) Delete(ctx context.Context, id uuid.UUID) error {
 	return nil
 }
 
+// List return all users
+func (m *GormUserRepository) List(ctx context.Context) ([]*User, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "user", "list"}, time.Now())
+	var rows []*User
+
+	err := m.db.Model(&User{}).Order("email").Find(&rows).Error
```

Comment:
Completely agree with you guys. I have the same concern. We need to use some sort of filter here instead which will limit the result up to a couple of dozen users or so I guess. But this PR is not about optimization. I just switched the existing /api/identities (has the same problem) to /api/users.
Let's switch to new User/Identity model + KC token first then address other issues here.